### PR TITLE
Add a docbuilder vagrant VM

### PR DESCRIPTION
**Pull request by crunchyjohn**
1) Added a docbuilder vagrant VM to convert README.md to pdf
2) Added some gitignores

### CODE REVIEW

File: docbuilder/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Note:  Vagrant Guest Additions is required
+# You may acquire it on your host system via: "sudo vagrant plugin install vagrant-vbguest"
```
Comment (mpalmi) [inline docbuilder/Vagrantfile:2]: Might want to include this in the README.txt

**Review by mpalmi — COMMENTED** [review]
Checked out and built on my dev machine. Looks fine to me (with one minor README suggestion).

File: docbuilder/Vagrantfile (hunk 1 above)
Comment (crunchyjohn) [reply docbuilder/Vagrantfile:2]: Thanks!  I have pushed that change

**Review by mpalmi — APPROVED** [review]
Looks good :+1:

**Review by jconway — APPROVED** [review]

**Pull request merged**